chatops4s-slack/src/main/scala/chatops4s/slack/instances/CatsMonadInstance.scala

```diff
@@ -0,0 +1,10 @@
+package chatops4s.slack.instances
+
+import cats.effect.{MonadCancelThrow, Sync}
+import chatops4s.slack.Monad
+
+given catsEffectMonadInstance[F[_]: Sync]: Monad[F] with {
+  def pure[A](value: A): F[A]                     = Sync[F].pure(value)
+  def flatMap[A, B](fa: F[A])(f: A => F[B]): F[B] = Sync[F].flatMap(fa)(f)
+  def raiseError[A](error: Throwable): F[A]       = Sync[F].raiseError(error)
+}
```

Comment:
This doesnt look right. We still have cats dependency (as a library). I suggest: 
1. for outbound, use sttp monad instance that you get from the backend
2. for inbound, you can leave it as IO for now, I need to think deeper on what a good approach is to generalize it